server_development/topics/themes-selector.adoc

```diff
@@ -0,0 +1,12 @@
+= Theme Selector
+
+By default the theme configured for the realm is used, with the exception of clients being able to override the login
+theme. This behavior can be changed through the Theme Selector SPI.
+
+This could for example be used to create to select different themes for desktop and mobile devices by looking at the
+user agent header.
+
```

Comment:
This could be used to create to select different themes for desktop and mobile devices by looking at the user agent header, for example.